docs/use-cases/static-web-content.md

```diff
@@ -0,0 +1,32 @@
+# I have Ceph.  So now what?  Application notes for Ceph cluster usage.
+
+Ceph has three primary client interfaces available: RGW, an Amazon S3
```

Comment:
It could be helpful to set up the application on a Genesis-built cluster, taking step-by-step notes. Those notes are the spine of the recipe.